cli/server/server.go

```diff
@@ -439,7 +439,8 @@ func mkP2PNotary(config network.ServerConfig, chain *core.Blockchain, serv *netw
 		Log:     log,
 	}
 	n, err := notary.NewNotary(cfg, serv.Net, serv.GetNotaryPool(), func(tx *transaction.Transaction) error {
-		if err := serv.RelayTxn(tx); err != nil {
+		err := serv.RelayTxn(tx)
+		if err != nil && err != core.ErrAlreadyExists {
```

Comment:
We need to use `errors.Is`, because `core.ErrAlreadyExists` may be prefixed by additional info, like here:
```
Apr 05 01:09:12 b neogo-morph-cn[40361]: 2022-04-05T01:09:12.475Z        ERROR        new transaction callback finished with error        {"error": "can't relay completed notary transaction: hash ca915660d581e3d469269bb3f8906efe99cbf4e1a7830d9b933530d2dda6aade, error: mempool: already exists"}
```

Reply:
It was meant to be written with `errors.Is` and somehow it's not, hmm.